tests/integration/test_streak_recalculation.py

```
@@ -0,0 +1,446 @@
+    journal_factory,
+):
+    """Case A: Deleting latest entry reduces streak."""
+    journal = journal_factory(title="Streak Test Journal")
+    token = api_user.access_token
+
+    base_date = date.today()
+    entry_date_20 = base_date - timedelta(days=2)
+    entry_date_21 = base_date - timedelta(days=1)
+    entry_date_22 = base_date
+
+    entry_20 = api_client.create_entry(
```

Ruff (0.14.5)

Local variable `entry_20` is assigned to but never used

Remove assignment to unused variable `entry_20`

(F841)